packages/config/src/bridges/lzOmnichain.ts

```diff
@@ -0,0 +1,182 @@
+import { Bridge } from './types'
+
+export const lzOmnichain: Bridge = {
+  type: 'bridge',
+  id: ProjectId('lzOmnichain'),
+  display: {
+    name: 'Omnichain (LayerZero)',
+    slug: 'lzOmnichain',
+    links: {
+      websites: ['https://layerzero.network/'],
+      repositories: [
+        'https://github.com/LayerZero-Labs/',
+        'https://github.com/stargate-protocol',
+      ],
+      socialMedia: ['https://twitter.com/StargateFinance'],
+    },
+    description:
+      'Omnichain tokens are built on top of LayerZero AMB protocol. They inherit its security and add on top of it their own additional security assumptions.',
+  },
+  riskView: {
+    validatedBy: {
+      value: 'Third Party',
+      description:
+        'Transfers need to be independently confirmed by oracle attesting to source chain checkpoints and Relayer providing merkle proof of the transfer event.',
+      sentiment: 'bad',
+    },
+    sourceUpgradeability: {
+      value: 'Yes',
+      description:
+        "Omnichain tokens can be individually upgradable and it's security assumptions must be individually assessed for each individual token.",
+      sentiment: 'bad',
+    },
+    destinationToken: RISK_VIEW.CANONICAL,
+  },
+  technology: {
+    destination: ['Various'],
+    category: 'Token Bridge',
+    principleOfOperation: {
+      name: 'Principle of operation',
+      description:
+        'Omnichain tokens are tokenized Token Bridges. One chain is designated as main and acts as an token escrow. Transfers from the main chain are done using typical lock-mint model. Transfers between\
+        other (non-main) chains are made using burn-mint model. The implementation details may vary between each individual omnichain token and must be individually assessed.',
+      risks: [],
+      references: [],
+    },
+    validation: {
+      name: 'Oracles and Relayers',
+      description:
+        'Omnichain tokens are built on top of LayerZero protocol. LayerZero relies on Oracles to periodically submit source chain block hashes to the destination chain.\
+        Once block hash is submitted, Relayers can provide the merkle proof for the transfers. Omnichain token owner can withdraw all funds from the bridge escrow\
+        by changing Oracle/Relayer pair.',
+      references: [],
+      risks: [
+        {
+          category: 'Users can be censored if',
+          text: 'oracles or relayers fail to facilitate the transfer.',
+          isCritical: true,
+        },
+        {
+          category: 'Funds can be stolen if',
+          text: 'oracles and relayers collude to submit fraudulent block hash and relay fraudulent transfer .',
+          isCritical: true,
+        },
+        {
+          category: 'Funds can be stolen if',
+          text: 'omnichain token owner changes Oracle/Relayer pair for their own.',
+          isCritical: true,
+        },
+      ],
+      isIncomplete: true,
+    },
+  },
+  config: {
+    escrows: [
```

Comment:
Note to someone. The escrow list should somehow be autofetched (similar maybe to multichain).

One way is to find trades going through LayerZero and find unique contracts